### PR TITLE
feat(withdrawal lock): implemented dynamic load of lock interval

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
this PR replace all static usage of `3 days` and add dynamic way of showing `n days` where n is something we obtain from API

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Comment by amadeo-bc** [comment]
@plondon do you think you could review this today? BE is ready to release a change that might break the FE and it'd be great to get this into prod. 

https://blockc.slack.com/archives/G01EH5KSYEP/p1605088641000100

File: packages/blockchain-wallet-v4-frontend/src/data/components/sendBch/sagas.ts [hunk 1]
```diff
@@ -436,7 +443,19 @@ export default ({
         )
         if (fromType === ADDRESS_TYPES.CUSTODIAL && error) {
           if (error === 'Pending withdrawal locks') {
-            yield put(actions.alerts.displayError(C.LOCKED_WITHDRAW_ERROR))
+            yield call(getWithdrawalLockCheck)
+            const rule = (yield select(
+              selectors.components.send.getWithdrawLockCheckRule
+            )).getOrFail({})
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/sendBch/sagas.ts:449]: `getOrElse()` // fallback to 3 days

File: packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts [hunk 2]
```diff
@@ -597,7 +604,19 @@ export default ({
         )
         if (fromType === ADDRESS_TYPES.CUSTODIAL && error) {
           if (error === 'Pending withdrawal locks') {
-            yield put(actions.alerts.displayError(C.LOCKED_WITHDRAW_ERROR))
+            yield call(getWithdrawalLockCheck)
+            const rule = (yield select(
+              selectors.components.send.getWithdrawLockCheckRule
+            )).getOrFail({})
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts:610]: make generic util saga in `packages/blockchain-wallet-v4-frontend/src/data/components/send/sagas.ts`

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts:610]: call for all coins

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx [hunk 3]
```diff
@@ -182,11 +188,11 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             style={{ marginRight: '8px' }}
           />
           <Text size='12px' weight={500} color='grey900'>
-            {props.order.paymentType === 'PAYMENT_CARD' ||
-            props.order.paymentType === 'USER_CARD' ? (
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx:186]: leave in `USER_CARD` check

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx [hunk 4]
```diff
@@ -283,9 +287,15 @@ const Success: React.FC<Props> = props => {
...
-                values={{ wallet: props.order.inputCurrency }}
+                id='modals.simplebuy.summary.complete_card_info_main'
+                defaultMessage='For your security, first time card purchases are subject to a {days} day holding period before you can send or withdraw your purchased crypto. We will notify you when the hold is lifted.'
+                values={{ days: days }}
+              />
+            </Text>
+            <Text color='grey600' size='14px' weight={500}>
+              <FormattedHTMLMessage
+                id='modals.simplebuy.summary.complete_card_info_additional'
+                defaultMessage="In the meantime, you can sell into Cash, swap, and trade within Blockchain.com.<a href='https://support.blockchain.com/hc/en-us/articles/360048200392' rel='noopener noreferrer' target='_blank'>Learn more.</a>"
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx:298]: remove `<a>` from text and add new formatttedMessage

File: packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/LockTimeTooltip/index.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    return data.cata({
+      Success: val => (
+        <LockTime
+          {...rest}
+          lockTime={moment
+            .duration(val.withdrawLockCheck.lockTime, 'seconds')
+            .days()}
+        />
+      ),
+      Failure: () => null,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/LockTimeTooltip/index.tsx:29]:
```
val => (
        <LockTime
          {...rest}
          lockTime={3}
        />
      ),
```

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts [hunk 6]
```diff
@@ -597,7 +603,7 @@ export default ({
         )
         if (fromType === ADDRESS_TYPES.CUSTODIAL && error) {
           if (error === 'Pending withdrawal locks') {
-            yield put(actions.alerts.displayError(C.LOCKED_WITHDRAW_ERROR))
+            yield call(showWithdrawalLockAlert)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts:606]: 👏

**Review by schnogz — APPROVED** [review]

**Pull request merged**